CHANGELOG.md

```diff
@@ -1,6 +1,139 @@
 Change Log
 ==========
 
+# Version 3.6.0
+
+_2022-09-01_
+
+This version brings initial support for `@defer` as well as data builders.
+
+## 💙️ External contributors
+
+Many thanks to @engdorm, @Goooler, @pt2121 and @StylianosGakis for their contributions!
+
+## ✨️ [new] `@defer` support
+
+`@defer` is [stage 1 GraphQL spec proposal](https://github.com/graphql/graphql-spec/blob/main/CONTRIBUTING.md#stage-1-proposal) to allow incremental delivery of response payloads. 
+
+`@defer` allows you to mark a fragment as deferrable, meaning it can be omitted in the initial response and delivered as a subsequent payload. This improves latency for all fields that are not in that fragment. You can read more about `@defer` in the [RFC](https://github.com/graphql/graphql-wg/blob/main/rfcs/DeferStream.md) and contribute/ask question in the [`@defer` working group](https://github.com/robrichard/defer-stream-wg).
```

Comment:
```suggestion
`@defer` allows you to specify a fragment as deferrable, meaning it can be omitted in the initial response and delivered as a subsequent payload. This improves latency for all fields that are not in that fragment. You can read more about `@defer` in the [RFC](https://github.com/graphql/graphql-wg/blob/main/rfcs/DeferStream.md) and contribute/ask question in the [`@defer` working group](https://github.com/robrichard/defer-stream-wg).
```